src/website/views/getting-started/index.pug

```diff
@@ -1,12 +1,87 @@
 ---
 title: Introduction
 order: 1
-description: Chi provides developers with a collection of HTML and CSS patterns to build fast, reusable, and consistent responsive interfaces.
-tabs: {"Chi Web Components" : "webcomponentstab", "Chi CSS" : "csstab"}
+homepage: true
+heroTitle: Build products customers love
+heroSubtitle: Chi is CenturyLink's design system for building consistent digital products and experiences.
 ---
 
-.chi-tabs-panel.-active#webcomponentstab
-  include ./introduction/_web-components
+.chi-grid.-py-lg--2.-py-xl--4
+    .chi-col.-w--12.-w-md--6.-w-lg--5.-mb--1.-mb-lg--6
+        .chi-grid.-no-gutter
+            .chi-col.-w--12.-p--2.-p-lg--0
+                h3.-text--h3.-mt--0.-mb--1.-mb-sm--2 Accessible
+                p.-text--large.-text--grey.-m--0
+                    | Chi components follow Section 508 and WCAG AA standards which ensure the best experience
+                    | is provided for all customers, including those with disabilities.
+    .chi-col.-w--12.-w-md--6.-w-lg--5.-o-lg--2.-mb--1.-mb-lg--6
+        .chi-grid.-no-gutter
+            .chi-col.-w--12.-p--2.-p-lg--0
+                h3.-text--h3.-mt--0.-mb--1.-mb-sm--2 Composable
+                p.-text--large.-text--grey.-m--0
+                    | Chi components are flexible, scalable building blocks that can be implemented as is or
+                    | assembled in various combinations to satisfy a diverse set of requirements.
+    .chi-col.-w--12.-w-md--6.-w-lg--5.-mb--1.-mb-lg--6
+        .chi-grid.-no-gutter
+            .chi-col.-w--12.-p--2.-p-lg--0
+                h3.-text--h3.-mt--0.-mb--1.-mb-sm--2 Reusable
+                p.-text--large.-text--grey.-m--0
+                    | Chi components provide tried-and-tested patterns that enable teams to ship faster, accrue
+                    | less technical debt, and deliver more predictable and seamless experiences to customers.
+    .chi-col.-w--12.-w-md--6.-w-lg--5.-o-lg--2.-mb--1.-mb-lg--6
+        .chi-grid.-no-gutter
+            .chi-col.-w--12.-p--2.-p-lg--0
+                h3.-text--h3.-mt--0.-mb--1.-mb-sm--2 Compatible
+                p.-text--large.-text--grey.-m--0
+                    | Chi's framework agnostic web components and independent CSS layer are compatible with any
+                    | major framework (AngularJS, ReactJS, VueJS) or no framework at all.
 
-.chi-tabs-panel#csstab
-  include ./introduction/_css
+.chi-h3.-mt--3.-d--none Getting Started
```

Comment:
@mattnickles is this going to be released in the 2.0 prod version or are just ideas?

Reply:
Just ideas. If more content was added I was going to render this at the bottom of the page to guide users where to go next.